.gitignore

```diff
@@ -14,4 +14,8 @@
 
 .vscode
 
-meshery-traefik-mesh
\ No newline at end of file
+meshery-traefik-mesh
```

Comment:
We need to remove the `meshery-traefik-mesh` and instead add `**errorutil` // @DelusionalOptimist

Reply:
Do we really need to? :thinking:
I'm not sure if they'll build the utility.